config.m4

```diff
@@ -1,35 +1,36 @@
-dnl $Id$
 dnl config.m4 for extension crypto
 
 PHP_ARG_WITH(crypto, for crypto support,
 [  --with-crypto             Include crypto support])
 
 if test "$PHP_CRYPTO" != "no"; then
-  test -z "$PHP_OPENSSL" && PHP_OPENSSL=no
-  if test "$PHP_OPENSSL" != "no" || test "$PHP_OPENSSL_DIR" != "no"; then
-    dnl Try to find pkg-config
-    if test -z "$PKG_CONFIG"; then
-      AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
-    fi
-    dnl If pkg-config is found try using it
-    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists openssl; then
+  dnl Try to find pkg-config
+  if test -z "$PKG_CONFIG"; then
+    AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
+  fi
+  dnl If pkg-config is found try using it
+  if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists openssl; then
+    PKG_CHECK_MODULES([OPENSSL], [openssl >= 1.1.1], [found_crypto_openssl=yes])
```

Comment:
This is breaking the builds on <7.4.

Reply:
Yeah there's been some changes in 7.4 build so there might be something extra that I need to do for lower branches. Will need to take a look later and figure it out.